f/medical/fn_famPassOut.sqf

```diff
@@ -0,0 +1,117 @@
+// F3 FA Medical
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// ====================================================================================
+
+// INITIAL VARIABLES 
+params ["_unit"];
+
+// ====================================================================================
+
+// SOUND EFFECTS
+private _sound = selectRandom [
+    "A3\Missions_F_EPA\data\sounds\WoundedGuyB_05.wss",
+    "A3\Missions_F_EPA\data\sounds\WoundedGuyB_06.wss",
+    "A3\Missions_F_EPA\data\sounds\WoundedGuyB_07.wss",
+    "A3\Missions_F_EPA\data\sounds\WoundedGuyB_08.wss",
+    "A3\Missions_F_EPA\data\sounds\WoundedGuyB_09.wss"
+];
+playSound3d [_sound,_unit];
+
+// ====================================================================================
+
+// RAGDOLL
+// If in a vehicle
+if (vehicle _unit != _unit) then {
+
+    if (driver vehicle _unit == _unit) then {
+        _anim = getArray (configfile >> "CfgMovesMaleSdr" >> "States" >> animationState _unit >> "interpolateTo");
+        _unit playMove (_anim select 0);
+    };
+
+    _unit setUnconscious true;
+
+} else {
+
+    // Ragdoll
+    _unit spawn f_fnc_famRagdoll;
+    
+};
+
+// ====================================================================================
+
+// INSTANT EFFECTS
+// Broadcast unit is unconscious.
+_unit setVariable ["FAM_CONSCIOUS",false,true]; 
+
+// If the unit is local and a player, remove their magazines (otherwise they can throw grenades while down)
+if(local _unit && isPlayer _unit) then
+{   
+
+    // keep a bandage and FAK in unit inventory so they can be picked up
+	if ("bandage" in magazines _unit) then {
+		_unit setVariable ["FAM_HASBANDAGE",true];  
+    }; 
+	if ("firstAidKit" in magazines _unit) then {
+		_unit setVariable ["FAM_HASFAK",true];  
+    }; 
+
+    _unit setVariable ["FAM_wound_down_mags",magazines _unit];
+    {
+        _unit removeMagazine _x;
+    } foreach magazines _unit;
+
+    _unit setVariable ["FAM_wound_down_items",(assignedItems _unit select {_x == "ItemGPS" || _x == "ItemMap"})];
+    {
+        _unit unassignItem _x;
+        _unit removeItem _x;
```

Comment:
no map or gps? 😰

Reply:
Only while you are passed out! Otherwise you can leave a million markers while unable to talk. I give ‘em back!